test/scripts/helpers/backbone/views/base.js

```diff
@@ -179,7 +186,7 @@ describe('backbone base view helper tests', function () {
         'images/social/logo.png');
       document.querySelectorAll('meta[property=\'og:image\']').length.should.equal(1);
       document.querySelector('meta[name=\'description\']').getAttribute('content').should.equal('second ' +
-        'description');
+        'description' + ' ' + 'second summary');
```

Comment:
@edwoodward This is a change from October. I have verified it did not get moved over to production branch.